client/src/components/BoulderCard.jsx

```diff
@@ -44,7 +53,8 @@ const BoulderCard = ({ boulder, detailedMode }) => {
             <StartPic src={boulderstart} alt="Boulder Start" />
             <Name>{name}</Name>
             <Likes>
-              <Heart style={{ width: '20px' }} />0
+              <Heart style={{ width: '20px' }} />
```

Comment:
Is there a reason why you're working with the style attribute and not with a styled component?

Reply:
The reason is that I just have this one line of code for styling in this case.

Reply:
I would prefer to work with a styled component even if it is only one style for better readability and consistency of your code. :) Therefore, I would recommend you to change this.

Reply:
Added to refactor list.